pipelines/manual-ci-kubernetes-pipeline.yml

```diff
@@ -1,5 +1,13 @@
 ---
 resources:
+
+- name: acceptance-tests-repo
+  type: git
+  source:
+    uri: git@github.com:ONSdigital/census-rm-acceptance-tests.git
+    private_key: ((github.service_account_private_key))
+    paths: [kubernetes.env, tasks/*]
```

Comment:
This seems to be our releases pipeline now, do we still want the acceptance tests in it?

Reply:
@AdamHawtin @davidmort said we're right to keep the acceptance tests in for now.